dist/jquery.flot.js

```diff
@@ -2170,7 +2170,7 @@ Licensed under the MIT license.
                 dec = -Math.floor(Math.log(delta) / Math.LN10);
 
             //if it is called with tickDecimals, then the precision should not be greather then that
-            if (dec > tickDecimals) {
+            if (tickDecimals && dec > tickDecimals) {
```

Comment:
When we zoomed in, if tickDecimals was null, dec became null and we got tickSize = 1. This caused some of the sub-unitary ticks to not be generated